examples/regression/1-quickstart/plot_compare_conformity_scores.py

```diff
@@ -0,0 +1,197 @@
+"""
+===========================================================
+Estimating prediction intervals of Gamma distributed target
+===========================================================
+This example uses :class:`mapie.regression.MapieRegressor` to estimate
+prediction intervals associated with Gamma distributed target.
+The limit of the absolute residual conformity score is illustrated.
+
+We use here the OpenML house_prices dataset:
+https://www.openml.org/search?type=data&sort=runs&id=42165&status=active.
+
+The data is modelled by a Random Forest model
+:class:`sklearn.ensemble.RandomForestRegressor` with a fix parameter set.
+The confidence intervals are determined by means of the MAPIE regressor
```

Comment:
Use "prediction intervals" instead of "confidence intervals".

Reply:
Done